src/BulletSoftBody/btDeformableBackwardEulerObjective.cpp

```diff
@@ -1,4 +1,6 @@
 /*
+ Written by Xuchen Han <xuchenhan2015@u.northwestern.edu>
+ 
```

Comment:
m_preconditioner is not deleted, please move destructor to c++ and delete there.

Reply:
fixed.